MapboxNavigation/FeedbackSubtypeViewController.swift

```diff
@@ -149,11 +149,12 @@ class FeedbackSubtypeViewController: FeedbackViewController {
 
     private func updateButtonTitle() {
         if selectedItems.count == 0 {
-            reportButton.setTitle(NSLocalizedString("NAVIGATION_REPORT_CANCEL", comment: "Title for button that cancels user's submission of feedback on navigation session issues."), for: .normal)
+            reportButton.setTitle(NSLocalizedString("NAVIGATION_REPORT_CANCEL", bundle: .mapboxNavigation, value: "Cancel", comment: "Title for button that cancels user's submission of feedback on navigation session issues."), for: .normal)
+            
         } else if selectedItems.count == 1 {
-            reportButton.setTitle(NSLocalizedString("NAVIGATION_REPORT_ISSUE", comment: "Title for button that submits user's feedback on navigation session issues."), for: .normal)
+            reportButton.setTitle(NSLocalizedString("NAVIGATION_REPORT_ISSUE", bundle: .mapboxNavigation, value: "Send Item", comment: "Title for button that submits user's feedback on navigation session issues."), for: .normal)
         } else {
-            reportButton.setTitle(NSLocalizedString("NAVIGATION_REPORT_ISSUES", comment: "Title for button that submits user's feedback on multiple navigation session issues."), for: .normal)
+            reportButton.setTitle(NSLocalizedString("NAVIGATION_REPORT_ISSUES", bundle: .mapboxNavigation, value: "Send Items",  comment: "Title for button that submits user's feedback on multiple navigation session issues."), for: .normal)
```

Comment:
This logic doesn’t work in some of the supported languages, such as Russian. We use a stringsdict to enable more flexible localization of grammatical plurals and genders:

https://github.com/mapbox/mapbox-navigation-ios/blob/6e54f9d38bb09e7cb1cdac7d194364e69f7d3ea8/MapboxNavigation/RatingControl.swift#L105-L109 https://github.com/mapbox/mapbox-navigation-ios/blob/dc9f1e9be68b18ca4f57882c98e0a58e85461507/MapboxNavigation/Resources/en.lproj/Localizable.stringsdict#L21-L36